src/aux_utils.cpp

```diff
@@ -739,15 +743,42 @@ namespace diskann {
         if (reorder_data_file_size != 8 + sizeof(float) *
                                               (size_t) npts_reorder_file *
                                               (size_t) ndims_reorder_file)
-          throw ANNException("Discrepancy in reorder data file size ", -1,
+          throw diskann::ANNException("Discrepancy in reorder data file size ", -1,
                              __FUNCSIG__, __FILE__, __LINE__);
       } catch (std::system_error &e) {
         throw FileException(reorder_data_file, e, __FUNCSIG__, __FILE__,
                             __LINE__);
       }
     }
 
-    // create cached reader + writer
+    //lorder -> Location to point/Id file
+    //porder -> Id to location file
+    unsigned npts_lorder, ndims_lorder, npts_porder, ndims_porder;
+    std::unique_ptr<char[]> lorder_data = std::make_unique<char[]>(npts_64);
```

Comment:
for load_bin, we dont need to initialize the array. the function will do it.. so just keep it as unique_ptr

Reply:
added unsigned pointer and passed it to the load_bin function